whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -340,9 +337,16 @@ private RdapObject mapCommonNoticesAndPort(final RdapObject rdapResponse, final
     private RdapObject mapCommonConformances(final RdapObject rdapResponse) {
         rdapResponse.getRdapConformance().addAll(List.of(RdapConformance.CIDR_0.getValue(),
             RdapConformance.LEVEL_0.getValue(), RdapConformance.NRO_PROFILE_0.getValue()));
+        mapRedactionConformance(rdapResponse);
         return rdapResponse;
     }
 
+    private void mapRedactionConformance(final RdapObject rdapResponse){
+        if (!rdapResponse.getRedacted().isEmpty() || (rdapResponse.getEntitySearchResults() != null && rdapResponse.getEntitySearchResults().stream().map(RdapObject::getRedacted).anyMatch(list -> !list.isEmpty()))){
```

Comment:
Correct me if I am wrong, even if redacted field is empty we will still see empty element in json response?

Reply:
nop, I sent you a json output from dev without redaction